jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -120,40 +121,45 @@ private static void completeInitialization() {
 
 		if (Boolean.getBoolean("ibm.java9.forceCommonCleanerShutdown")) { //$NON-NLS-1$
 			Runnable runnable = () -> {
-				CleanerShutdown.shutdownCleaner();
 				Thread threads[] = null;
```

Comment:
Please use normal array declaration syntax and remove the redundant initialization:
```
    Thread[] threads;
```